packages/itinerary-body/src/defaults/access-leg-description.tsx

```diff
@@ -22,6 +22,13 @@ export default function AccessLegDescription({
   style
 }: Props): ReactElement {
   const intl = useIntl();
+  // Replace the Vertex Type for BIKESHARE with VEHICLE as we do not know that
```

Comment:
This seems to not cause problems with OTP1 but are we sure? Do we know that OTP1 will never return "BIKESHARE"?

Reply:
OTP1 can return BIKESHARE, and this will replace it with VEHICLE. However that will cause the behavior to be consistent between OTP1 and OTP2. It's true that with OTP1 `BIKESHARE` actually does mean bike share, but we have no way (inside this function) of knowing whether it's OTP1 bikeshare or OTP2 vehicle rental.